hieradata/integration.yaml

```diff
@@ -31,6 +31,9 @@ govuk::apps::whitehall::basic_auth_credentials: "%{hiera('http_username')}:%{hie
 govuk::apps::whitehall::highlight_words_to_avoid: true
 govuk::apps::whitehall::enable_tagging_to_new_taxonomy: "yes"
 
+govuk_beat::hosts:
+  - "%{hiera('logstash_host')}:5566"
```

Comment:
@afda16 overriding it in the credentials repo is fine. This was me being bad at puppet